Parsers/Logstash/input-aws_s3-output-loganalytics.conf

```diff
@@ -0,0 +1,34 @@
+// Logstash configuration file to ingest custome json file
+// Last Updated Date: Nov 5, 2019
+//
+// Usage Instructions:
+// Follow the wiki article for installation and usage Instructions.
+// https://github.com/Azure/Azure-Sentinel/wiki/Ingest-Custom-Logs-LogStash
+//
+// Parser Notes:
+// 1. Make sure the log type name is specified as alpha characters- no digits or special characters allowed.
```

Comment:
Probably remove the "1."  There is not really a list here.  Also, remove the hyphen and simply use a comma or period.

Reply:
corrected